www-client/seamonkey/files/seamonkey-2.53.10.2-ownertab.patch

```diff
@@ -0,0 +1,248 @@
+# HG changeset patch
```

Comment:
Where possible, add links to upstream bugs / commits / references so we know what a patch is for, when we can get rid of it, and where it came from.